sotodlib/preprocess/pcore.py

```diff
@@ -292,7 +292,11 @@ def _expand(new, full, wrap_valid=True):
                 assert tuple(new._assignments[k]) == ('dets', 'samps')
                 out[k] = _reform_csr_array(v, oidx, nidx, out[k].shape)
             else:
-                out[k][oidx] = v[nidx]
+                try:
```

Comment:
This conflicts with what's in main. Please merge main, resolve the merge conflicts and re-request review.

Reply:
There are not merge conflicts, I needed to add this because there were _arrays_ of scalars, not just scalars, causing `TypeError`.